Kotlin/Maths/Armstrong.kt

```diff
@@ -0,0 +1,51 @@
+/*Kotlin Program to check Given number is an Armstrong Number or not*/
+import java.util.*
+
+// Function for check Armstrong Number
+fun isArmstrong(num: Int): Boolean {
+    var isArmNum = false
+    var result : Int= 0
+    var temp = num
+
+    // Count No Digits in numbers
+    val digits = temp.toString().length
+
+    while (temp > 0) {
+        val r = temp % 10
```

Comment:
you should rename `r` as remainder just as to make it more readable

Reply:
@gat786 changes done. Please review.